### DISCUSSION TITLE
Domain with radio-scanner

### DISCUSSION THREAD

**Discussion by MississippiDev** [chuot/rdio-scanner#374] [Q&A]
I don’t know have I can say this but… I am trying to setup my domain for Rdio-scanner with the app and the instructions are really hard to understand

can anybody help me, please, or explain it to me?

**Comment by Clark985** [comment]
Still need help with this?
Basically, I'm port forwarding TCP 3000 from the Internet to the internal IP of my machine running Rdio-scanner.
If you have a domain name, you then need to create an A record pointing to your _public_ IP address.   For example,  rdio.mydomain.net to 1.2.3.4

Hit the website ipecho.net to determine what your public IP is.

I have main Rdio instance on an isolated vlan for extra security, though most consumer routers can't do that.

**Comment by repairfox** [comment]
I use packet riot service. I have very little programming/networking knowledge, and that was easy to implement.
